FP-Erlang/Week 6/Assignment 5/Nachiket/assign5.erl

```diff
@@ -0,0 +1,124 @@
+%
+%
+%  Assignment 5
+%
+%  Author- Nachiket
+%
+%
+%
+-module(assign5).
+-compile(export_all).
+-import(math,[pow/2]).
+
+%
+%  Q1) Currying In Erlang
+%     i) add()
+%     ii) subtract()
+%     iii) multiply()
+%     iV)  divide()
+
+add(X) -> 
+    AddX = fun(Y) -> X + Y end,
+    AddX.
+
+subtract(X)->
+	SubX = fun(Y) -> X-Y end,
+	SubX.
+
+multiply(X)->
+	MulX = fun(Y) -> X*Y end,
+	MulX.
+
+divide(X)->
+	DivX = fun(Y) -> X/Y end,
+	DivX.
+
```

Comment:
Add two argument versions of the above functions by making calls to the curried functions